### ISSUE TITLE
rpmlint prints warning if /usr/bin/python is not present

### ISSUE THREAD

**Issue by bkabrda** [rpm-software-management/rpmlint#111]
Hi, I have a mock chroot without /usr/bin/python (fedora-27-x86_64) and rpmlint is printing this warning on every run:

```
sh: /usr/bin/python: No such file or directory
```

I tracked this down to this line: https://github.com/rpm-software-management/rpmlint/blob/39467b6962f1b105e3d23c25511c2e74bf8295f0/TagsCheck.py#L438 - the problem seems to be that rpmlint tries to expand the `%python_sitearch` macro, but that fails because `/usr/bin/python` is not installed and the `rpm.expandMacro` function prints this warning. I think that this should either be silenced or rpmlint should depend on Python 2 - I don't understand rpmlint enough to be able to tell which of these solutions is more appropriate (or if there's third solution).

**Comment by praiskup** [comment]
A bit historical report:
https://bugzilla.redhat.com/show_bug.cgi?id=1243292

**Comment by scop** [comment]
Yep, the bugzilla report still describes the situation and my position on it accurately: possibly a bug, but not one in rpmlint.

**Comment by praiskup** [comment]
IOW, this is issue in rpm in macros, not rpmlint.  As discussed in the bug, rpm should (a) either not define the %python_site* macros if python is not installed, or (b) have those macros hard-wired, not evaluated with %() or (c) at least the error output should be silenced by 2>/dev/null.
